Packages/MIES/MIES_Browser_Plotter.ipf

```diff
@@ -484,7 +484,7 @@ Function CreateTiledChannelGraph(string graph, WAVE config, variable sweepNo, WA
 							unit = "a.u."
 						endif
 
-						axisLabel = "\Z08" + traceType + "\r(" + unit + ")"
+						axisLabel = "\Zr060" + traceType + "\r(" + unit + ")"
```

Comment:
@t-b The scaling is nice. Can it start with a larger font? - The axis label is too small when reducing the size of the sweep browser to the minimum allowable size.
![image](https://github.com/user-attachments/assets/58ddf966-efc9-4442-8c97-21950bb8cbe9)

Reply:
I've switched to 80% from 60%.